client/src/Pages/Main.js

```diff
@@ -8,19 +8,19 @@ import MainPopupController from "../Components/PopupController/PopUpController";
 
 const Main = () => {
   
-  /*
-  const someData = [
-    'int j = 20;',
-    'for (int i = 0; i < 10; i++) {',
-    '	System.out.print(j + " - " + i);',
-    'if (i > j)',
-    '		System.out.println(" < 0");',
-    '	else',
-    '		System.out.println(" > 0");',
-    '	j -= 1;',
-    '}'
-  ]
-*/
+  
+  // const someData = [
+  //   'int j = 20;',
+  //   'for (int i = 0; i < 10; i++)',
+  //   '	System.out.print(j + " - " + i);',
+  //   // 'if (i > j)',
+  //   // '		System.out.println(" < 0");',
+  //   // '	else',
+  //   // '		System.out.println(" > 0");',
+  //   // '	j -= 1;',
+  //   // '}'
+  // ]
```

Comment:
Is this comment needed?

Reply:
Not the double comments, but we should keep in the alternate default snippet datas because they are useful for debugging.